config/site_settings.yml

```diff
@@ -2348,6 +2348,10 @@ uncategorized:
     default: false
     hidden: true
 
+  allow_bulk_invite:
+    default: false
```

Comment:
One last thing: I think we should start with `default: true` here for now, because otherwise this is essentially removing a feature (and it will generate lots of support requests). Will also discuss internally...